### PR TITLE
update `external_ipv6_prefix` to not be output only

### PR DESCRIPTION
**Pull request by BBBmau**
This addresses the resource `google_compute_subnetwork` where the attribute `externalIpv6Prefix` is set as Ouput only, however when looking at the Google Docs it's not an output attribute. It's also worth mentioning that it has a Default value of `"2600:1900:4000:1ae::/64"` which is not mentioned on the [Google Cloud docs](https://cloud.google.com/compute/docs/reference/rest/v1/subnetworks#:~:text=for%20internal%20use.-,externalIpv6Prefix,-string)

**Release Note Template for Downstream PRs (will be copied)**

```release-note:bug
compute: changed `external_ipv6_prefix` field on `google_compute_subnetwork` resource to not be output only
```

### CODE REVIEW

**Review by NickElliot — CHANGES_REQUESTED** [review]
Could you include the external_ipv6_prefix in the update tests to verify the functionality is stable if update support is being added for it? Thanks!

**Comment by BBBmau** [comment]
Been attempting to add it to the update function but im finding this issue that was not present before. Could use some input on this since i would've expected the same error to appear when updating the resource with the other configurations.

=== CONT  TestAccComputeSubnetwork_update
    vcr_utils.go:152: Step 5/6 error: Error running apply: exit status 1
        
        Error: Error updating Subnetwork "projects/hc-terraform-testing/regions/us-central1/subnetworks/tf-test-4ljmmmy06d": googleapi: Error 400: Required field 'resource.fingerprint' not specified, required
        
          with google_compute_subnetwork.network-with-private-google-access,
          on terraform_plugin_test.tf line 7, in resource "google_compute_subnetwork" "network-with-private-google-access":
           7: resource "google_compute_subnetwork" "network-with-private-google-access" {
        
--- FAIL: TestAccComputeSubnetwork_update (198.07s)

docs says that the fingerprint attribute is required when updating the resource, which is odd since its not used at all.
https://cloud.google.com/compute/docs/reference/rest/v1/subnetworks#:~:text=Fingerprint%20of%20this,base64%2Dencoded%20string.

**Comment by BBBmau** [comment]
I'll need some more input on this, I went ahead and manually added the `external_ipv6` attribute in update3 and found this error. Though I am unsure what format is expecting since this is the default value

```
│ Error: Error creating Subnetwork: googleapi: Error 400: Invalid value for field 'resource.externalIpv6Prefix': '2600:1900:41a0:4ec9:0:0:0:0/64'. Specified external IPv6 address is not allocated to the project or does not belong to the specified scope., invalid
```

I also attempted to use `google_compute_global_address` with it in hopes that it would solve the error but no luck with that.

**Comment by NickElliot** [comment]
Could you push the test config you were using with global address? Thanks

**Comment by BBBmau** [comment]
sure thing @NickElliot

```hcl
provider "google" {
  zone = "us-west1-a"
}

resource "google_compute_network" "custom-test" {
  project = "hc-terraform-testing"
  name                    = "test"
  auto_create_subnetworks = false
}

resource "google_compute_global_address" "ipv6" {
  project = "hc-terraform-testing"
  name = "prefix"
  ip_version = "IPV6"
}

resource "google_compute_subnetwork" "subnetwork" {
  project = "hc-terraform-testing"
  name          = "testtest"
  ip_cidr_range    = "10.0.0.0/16"
  region           = "us-central1"
  network          = google_compute_network.custom-test.self_link
  stack_type       = "IPV4_IPV6"
  ipv6_access_type = "EXTERNAL"
  external_ipv6_prefix = "2600:1900:41a0:4ec9:0:0:0:0/64"
}
```

File: mmv1/products/compute/Subnetwork.yaml [hunk 1]
```diff
@@ -392,9 +392,9 @@ properties:
       The internal IPv6 address range that is assigned to this subnetwork.
   - !ruby/object:Api::Type::String
     name: 'externalIpv6Prefix'
-    output: true
+    default_from_api: true
     description: |
-      The range of external IPv6 addresses that are owned by this subnetwork.
+      The range of external IPv6 addresses that are owned by this subnetwork. Defaults to "2600:1900:4000:1ae::/64".
```
Comment (rileykarson) [inline mmv1/products/compute/Subnetwork.yaml:397]:
```suggestion
      The range of external IPv6 addresses that are owned by this subnetwork.
```

File: mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.erb [hunk 2]
```diff
@@ -605,23 +630,6 @@ resource "google_compute_subnetwork" "network-with-private-secondary-ip-ranges"
 `, cnName, subnetworkName)
 }
 
-func testAccComputeSubnetwork_secondaryIpRanges_update4(cnName, subnetworkName string) string {
```
Comment (rileykarson) [inline mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.erb:608]: I think a recent commit removed the wrong configuration! We want to remove the diff in this file entirely.

Comment (BBBmau) [reply mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.erb]: my mistake! should be good now.

**Review by rileykarson — APPROVED** [review]
Validated offline

**Pull request merged**